CRFModuleValidation/Background_Survey.json

```diff
@@ -93,6 +95,157 @@
                           "text": "Masters or Doctoral Degree (PhD, MD, JD, etc)",
                           "value": "masters or doctoral"
                           }]
+              },
+              {
```

Comment:
Just going to assume all the text in here is copypasted from the actual survey and not proofread it....

Reply:
yup. *sigh*